src/Calendar/docs/ExampleStandard.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import Calendar from 'wix-style-react/Calendar';
 import ToggleSwitch from 'wix-style-react/ToggleSwitch';
-import Label from 'wix-style-react/Label';
+import { Text } from '../../index';
```

Comment:
```suggestion
	import Text from 'wix-style-react/Text';
```

why importing from `index` file?